### DISCUSSION TITLE
AI Powered Chat Bot App with React Native

### DISCUSSION THREAD

**Discussion by anilyavas** [notJust-dev/notjustdev#233] [Ideas]
Since AI is popular and usage is significantly expanding on all sectors, why don't we build some chat bot application with React Native and OpenAI for the AI part of it and maybe voice recognition could be used with react-native-voice

**Comment by VadimNotJustDev** [comment]
Have you seen these:
- https://www.youtube.com/watch?v=D-hDcs6crQE
- https://www.youtube.com/watch?v=XNe0IO_p3sc

**Reply by anilyavas** [reply]
Yes i have seen them on DEVember and they were pretty well explained but i thought maybe a bigger and better independent application might be useful instead of part of DEVember.
